charts/stirling-pdf/README.md

```diff
@@ -40,8 +40,10 @@ helm repo add stirling-pdf https://stirling-tools.github.io/Stirling-PDF-chart
 | envsFrom | list | `[]` | Environment variables from secrets or configmaps to add to the stirling-pdf pods |
 | extraArgs | list | `[]` |  |
 | image.pullPolicy | string | `"IfNotPresent"` |  |
-| image.repository | string | `"frooodle/s-pdf"` |  |
-| image.tag | string | `nil` |  |
+| image.registry | string | `"ghcr.io"` |  |
+| image.repository | string | `"stirlingtools/stirling-pdf"` |  |
```

Comment:
ghcr.io repo is stirling-tools/stirling-pdf  with a ``-``

Our dockerhub is stirlingtools/stirling-pdf without a ``-`` since docker doesn't allow orgs with ``-``